needle/modules/storage/data/keychain_dump.py

```diff
@@ -32,23 +35,54 @@ def module_pre(self):
     # ==================================================================================================================
     def module_run(self):
         # Composing the command string
-        cmd = '{}'.format(self.device.DEVICE_TOOLS['KEYCHAINDUMPER'])
+        cmd = '{}'.format(self.device.DEVICE_TOOLS['KEYCHAIN_DUMP'])

```

Comment:
Do you mean the user's home directory on the device? 

If so, it doesn't look like it is possible to pass a path to keychain_dump. Is there a built-in way in needle of changing the working directory on the device other than sending mkdir blah and then cd blah to exec_command?